### PR TITLE
#8933 Add languages to dropdown menu

### LINKED ISSUE #8933: Add languages to dropdown menu
Add the following languages to the language menu:

- Català (Catalan)
- Hrvatski (Croatian)
- తెలుగు (Telugu)

### PR DESCRIPTION
**Pull request by lequoctrung-it**
Fixes #8933 

#### Short description of what this resolves:
Add more language options:
- Català (Catalan)
- Hrvatski (Croatian)
- తెలుగు (Telugu)

#### Changes proposed in this pull request:

- Bring translation files (.po) from back-end repository to front-end, then add options to the l10n service.

image

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/services/l10n.js [hunk 1]
```diff
@@ -32,7 +32,10 @@ export default class L10nService extends L10n {
       'th'      : 'ไทย',
       'vi'      : 'Tiếng Việt',
       'zh_Hans' : '中文（简体)',
-      'zh_Hant' : '中文（繁體)'
+      'zh_Hant' : '中文（繁體)',
```
Comment (norbusan) [inline app/services/l10n.js:35]: Wasn't there a discussion to use `zh_TW` and `zh_CN` etc or similar instead of `zh_Hans` which is rather uncommon?

**Review by mariobehling — CHANGES_REQUESTED** [review]
Please sort it alphabetically according to Latin script. Catalan under C. Hrvatski under H. Telegu under T.

File: app/services/l10n.js (hunk 1 above)
Comment (lequoctrung-it) [reply app/services/l10n.js:35]: Yeah, but this issue is handled in another place, you can check it in my previous PR.

**Review by norbusan — APPROVED** [review]

**Comment by lequoctrung-it** [comment]
> Please sort it alphabetically according to Latin script. Catalan under C. Hrvatski under H. Telegu under T.

Oke I will update it later tonight.

File: .yarnrc
Comment (norbusan) [inline .yarnrc:1]: Please remove this file, it is auto-generated and should not be checked in

File: .yarn/releases/yarn-1.22.19.cjs
Comment (norbusan) [inline .yarn/releases/yarn-1.22.19.cjs:1]: Please remove this file

**Review by norbusan — CHANGES_REQUESTED** [review]

**Review by norbusan — APPROVED** [review]

**Pull request merged**